internal/server/config/config.go

```diff
@@ -23,10 +23,11 @@ const (
 	DefaultMaxConcurrentReadsForCheck       = math.MaxUint32
 	DefaultMaxConcurrentReadsForListObjects = math.MaxUint32
 
-	DefaultWriteContextByteLimit = 32 * 1_024 // 32KB
-	DefaultCheckQueryCacheLimit  = 10000
-	DefaultCheckQueryCacheTTL    = 10 * time.Second
-	DefaultCheckQueryCacheEnable = false
+	DefaultWriteContextByteLimit      = 32 * 1_024 // 32KB
+	DefaultCheckQueryCacheLimit       = 10000
+	DefaultCheckQueryCacheTTL         = 10 * time.Second
+	DefaultCheckQueryCacheEnable      = false
+	DefaultMaxConditionEvaluationCost = 100 // care should be taken here - decreasing can cause API compatibility problems with Conditions
```

Comment:
In this case, will it make sense to have smaller default max condition evaluation cost and increase on an as-needed basis?

Reply:
I see that 100 is used.  So never mind.